relay-server/src/services/store.rs

```diff
@@ -189,13 +189,15 @@ impl StoreService {
         let retention = envelope.retention();
         let event_id = envelope.event_id();
 
+        let use_feedback_ingest_v2 = self.global_config.current().options.feedback_ingest_v2;
```

Comment:
not sure what this naming implies, maybe just `separate_attachments_from_feedback`?

Reply:
Decided to name it this because it also results in some code restructuring / different code path for producing feedback in StoreService.

I agree the naming's a little funky though, will sleep on it + see what Vienna thinks?

Reply:
I would also prefer aligning the variable name (and the field name `feedback_ingest_v2`) with either the feature name ("inline attachments") or renaming it to describe what it actually enables.

Reply:
The flag's docs say it's temporary. Is it only used for testing in development or should we update the docs?

Reply:
It's used like a feature flag - we'll set it to true in test region(s), test the SDK, and remove this logic + non-v2 code path once satisfied. The flag is just to quickly "revert" in case things start to break

Reply:
What did you have in mind for updating the docs?